docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,28 @@
 Introduction to Workflows, Tools, and Services
 ==============================================
 
+Tools and workflows hosted on Dockstore are computational steps described in languages that make it easy for their reuse in different compute environments. Generally, descriptor languages request that a user include a containerized compute environment (such as a Docker image) that packages up all of the code and dependencies needed to complete the computational step.
+
+Not all descriptor languages make a distinction between a tool and a workflow. Most users will register workflows on Dockstore. 
+
 What is a workflow?
 -------------------
 
-In general terms, a workflow is a chain of commands that has several steps linked together, which create some sort of final output. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
+In general terms, a workflow describes a chain of commands in a specific order. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
 
-vcf file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
+VCF file --> call tool convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
```

Comment:
I'm in favor of the capitalization of VCF but 
> call tool convert to GDS file

Is not what's happening unless you mean "tool" to mean any arbitrary program, which conflicts the definition of tool we give a few lines later.

Reply:
I agree with this. I think we can keep the original way this was written, plus the capitalization of VCF.